src/runner/mod.rs

```diff
@@ -96,6 +78,20 @@ impl Runner {
             }
         }
     }
+
+    fn prepare_before_run(&self) {
+        panic::set_hook(Box::new(|_panic_info| {
+            // XXX panics already catched at the test call site, don't output the trace in stdout
+        }));
+        let threads = if self.configuration.parallel { 0 } else { 1 }; // 0 is rayon default
+        let rayon_config = rayon::Configuration::new().num_threads(threads);
+        let _ = rayon::initialize(rayon_config);
```

Comment:
By moving the block within `par_iter()` into a dedicated method we should be able to simply use an `if self.configuration.parallel { … } else { … }`. 🤔